api/v1alpha1/common_validation.go

```diff
@@ -63,22 +63,11 @@ func ValidateScheduler(schedulerObject InnerSchedulerObject, spec *field.Path) f
 func validateSchedulerParams(duration *time.Duration, durationField *field.Path, spec *SchedulerSpec, schedulerField *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if duration != nil && spec != nil {
-
 		cronField := schedulerField.Child("cron")
-		scheduler, err := ParseCron(spec.Cron, cronField)
+		_, err := ParseCron(spec.Cron, cronField)
 		if len(err) != 0 {
 			allErrs = append(allErrs, err...)
 		}
-
-		if scheduler != nil {
-			tmpTime := time.Time{}
-			nextTime := scheduler.Next(tmpTime)
-			interval := nextTime.Sub(tmpTime)
-			if *duration >= interval {
-				allErrs = append(allErrs, field.Invalid(cronField, spec.Cron,
-					fmt.Sprintf("the scheduling interval:\"%s\" must be greater than the duration:%s", spec.Cron, *duration)))
-			}
-		}
```

Comment:
Why remove the check, seems that we can get the `interval` in another way(get the next time t1, and the next time t2 of t1, interval = t2 - t1)?

Reply:
https://github.com/chaos-mesh/chaos-mesh/pull/1286#issuecomment-747232617
We talked about it before 🤣

Reply:
🥵